codeval.py

```diff
@@ -54,7 +33,7 @@ def __init__(self):
             user = self.canvas.get_current_user()
             info(f"connected to canvas as {user.name} ({user.id})")
         except:
-            error(f"there was a problem accessing canvas.")
+            error(f"there was a problem accessing canvas.", True)
```

Comment:
why is there a Turue?

Reply:
Removed True and replaced with errorWithException()